draft-hsts-tracking-prevention.md

```diff
@@ -0,0 +1,256 @@
+---
+###
+# Internet-Draft Markdown Template
+#
+# Rename this file from draft-todo-yourname-protocol.md to get started.
+# Draft name format is "draft-<yourname>-<workgroup>-<name>.md".
+#
+# For initial setup, you only need to edit the first block of fields.
+# Only "title" needs to be changed; delete "abbrev" if your title is short.
+# Any other content can be edited, but be careful not to introduce errors.
+# Some fields will be set automatically during setup if they are unchanged.
+#
+# Don't include "-00" or "-latest" in the filename.
+# Labels in the form draft-<yourname>-<workgroup>-<name>-latest are used by
+# the tools to refer to the current version; see "docname" for example.
+#
+# This template uses kramdown-rfc: https://github.com/cabo/kramdown-rfc
+# You can replace the entire file if you prefer a different format.
+# Change the file extension to match the format (.xml for XML, etc...)
+#
+###
+title: "HSTS Tracking Prevention"
+category: info
+
+docname: draft-hsts-tracking-prevention-latest
+submissiontype: IETF  # also: "independent", "editorial", "IAB", or "IRTF"
+number:
+date:
+consensus: true
+v: 3
+area: AREA
+workgroup: WG Working Group
+keyword:
+ - HSTS
+ - tracking prevention
+venue:
+  group: WG
+  type: Working Group
+  mail: WG@example.com
+  arch: https://example.com/WG
+  github: USER/REPO
+  latest: https://example.com/LATEST
+
+author:
+ -
+    fullname: Steven Bingler
+    organization: Google
+    email: bingler@google.com
+
+normative:
+  RFC6767
+
+informative:
```

Comment:
```
*** bad section normative: "RFC6767"
/usr/lib/ruby/gems/3.3.0/gems/kramdown-rfc2629-1.7.24/lib/kramdown-rfc/command.rb:361:in `block in xml_from_sections': undefined method `each' for an instance of String (NoMethodError)
```

Looking at https://github.com/cabo/kramdown-rfc/blob/master/lib/kramdown-rfc/command.rb#L358-L361

Is this barfing because of the empty string for `informative`? Is it valid to just delete that key?

Reply:
That seems reasonable, worth a shot.

Reply:
Seems reasonable, let's try it.